NppNavigateTo/Forms/FrmNavigateTo.cs

```diff
@@ -596,35 +599,38 @@ private void dataGridFileList_CellPainting(object sender, DataGridViewCellPainti
                                 int indexOfN = gridCellValue.IndexOf("\n", StringComparison.CurrentCultureIgnoreCase);
                                 Rectangle hlRect = new Rectangle();
                                 hlRect.Y = e.CellBounds.Y + 2;
-
-                                String sBeforeSearchword = gridCellValue.Substring(0, startIndexInCellValue);
-                                String sSearchWord = gridCellValue.Substring(startIndexInCellValue, word.Trim().Length);
-                                Size s1 = TextRenderer.MeasureText(e.Graphics, sBeforeSearchword, e.CellStyle.Font,
-                                    e.CellBounds.Size, TextFormatFlags.TextBoxControl);
-                                Size s2 = TextRenderer.MeasureText(e.Graphics, sSearchWord, e.CellStyle.Font,
+                                string sBeforeSearchWord = gridCellValue.Substring(0, startIndexInCellValue);
+                                Size sizeBeforeSearchWord = TextRenderer.MeasureText(e.Graphics, sBeforeSearchWord, e.CellStyle.Font,
                                     e.CellBounds.Size, TextFormatFlags.TextBoxControl);
-                                if (s1.Width > 5)
+                                if (IsFuzzyResult)
+                                {
+                                    // if not fuzzy, the text to be highlighted is just the original word
+                                    string sSearchWord = gridCellValue.Substring(startIndexInCellValue, trimmedWord.Length);
```

Comment:
In comment you mentioned `if not fuzzy` but in code you check `IsFuzzyResult`. Please could you clarify it a bit or change a comment?